public/npm-debug.log

```diff
@@ -0,0 +1,47 @@
+0 info it worked if it ends with ok
```

Comment:
I would throw `npm-debug.log` into the `.gitignore`

https://github.com/github/gitignore/blob/master/Node.gitignore